### ISSUE TITLE
Relying on reporter configuration that might be missing

### ISSUE THREAD

**Issue by kpiwko** [arquillian/arquillian-governor#5]
ReporterConfiguration might not be defined here:

https://github.com/arquillian/arquillian-governor/blob/0414ad3ca53d1d0e24142b6e07e0b985311ab705/skipper/src/main/java/org/arquillian/extension/governor/skipper/impl/SkipperReporter.java#L65

**Comment by smiklosovic** [comment]
https://github.com/arquillian/arquillian-governor/commit/c0270643235a6f36fec32ca7b4c0a5d3d4ac52f3

**Comment by smiklosovic** [comment]
even we do not have repository element in the pom.xml, we are relying on an artifact which is Maven Central but that artifact contains repository element itself, the same one we just deleted hence we just delegated this problem to the external party and we effectively solved nothing.

The proper solution would be to release Jira connector artifacts from Atlassian to Maven Central directly however the result of this request is questionable and out of scope of the alpha2 release because of the lack of time.

**Comment by smiklosovic** [comment]
Even we do not have the repository element in the `pom.xml`, we are relying on an artifact which is in Maven Central but that artifact contains repository element itself, the same one we just deleted hence we just delegated this problem to the external party and we effectively solved nothing.

The proper solution would be to release Jira connector artifacts from Atlassian to Maven Central directly however the result of this request is questionable and out of scope of the alpha2 release because of the lack of time.

I leave this open for tracking purposes.
